io/net/infiniband/ib_pingpong.py

```diff
@@ -104,7 +104,7 @@ def setUp(self):
         self.peer_iface = ''
         cmd = "ip addr show"
         output, exitcode = self.run_command(cmd)
-        for line in output.splitlines():
+        for line in str(output).splitlines():
```

Comment:
decode('utf-8') instead of str() please

Reply:
i tried with decode('utf-8') but i am geeting error," ERROR: 'list' object has no attribute 'decode' " so i kept like this.

Reply:
you need to decode each element in the list